sdk/servicebus/service-bus/test/atomManagement.spec.ts

```diff
@@ -560,8 +641,12 @@ const alwaysBeDeletedRule = "alwaysbedeletedrule";
 });
 
 // Topic tests
+// Create different topic each testcase as updates to same topic/subscription does not scale on service side and gives us error -
```

Comment:
Appreciate the comment!

Reply:
Thanks and yes! This is actually one other reason why I would refrain from the afterEach beforeEach approach - because the service may not be able to handle.

This was quite a mess to deal with leading to specific test setup such as above.

So, the more I think about it - it may be very hard to achieve the level of test isolation we are looking for. I'll do one pass of test refactoring to see if things can be improved without breaking.

Reply:
See https://github.com/Azure/azure-sdk-for-js/pull/6112#issuecomment-555716170